runtime/Jakt/Platform.h

```diff
@@ -42,7 +42,7 @@
 #ifdef ALWAYS_INLINE
 #    undef ALWAYS_INLINE
 #endif
-#define ALWAYS_INLINE
+#define ALWAYS_INLINE __attribute__((always_inline)) inline
```

Comment:
This was intentionally removed I think. Why is it needed?

Reply:
This error is from:

runtime/Jakt/Memory.h:19:20: error: no previous declaration for 'void fast_u32_copy(u32*, const u32*, size_t)' [-Werror=missing-declarations]
ALWAYS_INLINE void fast_u32_copy(u32* dest, u32 const* src, size_t count)

runtime/Jakt/Memory.h:30:20: error: no previous declaration for 'void fast_u32_fill(u32*, u32, size_t)' [-Werror=missing-declarations]
ALWAYS_INLINE void fast_u32_fill(u32* dest, u32 value, size_t count)